resources/icons/ui/arrow-down.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M16 3C16.5523 3 17 3.44772 17 4V25.5858L25.2929 17.2929C25.6834 16.9024 26.3166 16.9024 26.7071 17.2929C27.0976 17.6834 27.0976 18.3166 26.7071 18.7071L16.7071 28.7071C16.3166 29.0976 15.6834 29.0976 15.2929 28.7071L5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929C5.68342 16.9024 6.31658 16.9024 6.70711 17.2929L15 25.5858V4C15 3.44772 15.4477 3 16 3Z" fill="#212121"/>
+</svg>
```

Comment:
This is used for downloads, right? Could you use the download icon instead?

Reply:
Good point